src/js/client/canHandleStrings.js

```diff
@@ -0,0 +1,57 @@
+// Convenience grade to deal with APIs (like express-couchUser) that return JSON as a string.  The only expected use
+// of this is in combination with an `ajaxCapable` grade, which has the rest of the wiring that uses the options and
+// invokers outlined here.  Instantiating this grade by itself will not work.
+//
+/* global fluid */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.express.user.frontend.canHandleStrings");
+
+    gpii.express.user.frontend.canHandleStrings.stringToJSON = function (that, data) {
+        if (typeof data === "string") {
```

Comment:
I suspect this grade was covering for earlier crappiness in bad AJAX responses either from the previous user management library, from my pre `gpii-express` API code, or from both.

I was able to remove this grade entirely and all tests pass, so all client-side components in this package now accept only JSON data in responses and fail if they don't receive it.